flink-connectors/flink-sql-connector-hbase/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,13 @@
+flink-sql-connector-hbase
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.hbase:hbase-server:1.4.3
+- org.apache.zookeeper:zookeeper:3.4.10
+- org.apache.htrace:htrace-core:3.1.0-incubating
```

Comment:
Looks like here we missed many dependencies.

> All bundled dependencies must be listed in NOTICE, grouped by license type, using standard maven syntax (groupId:artifactId:version). 

[1] https://cwiki.apache.org/confluence/display/FLINK/Licensing

Reply:
I have checked the LICENSE and looks good to me. But would be better if @rmetzger  can verify it again.